index.html

```diff
@@ -54,24 +54,24 @@
       <div class="container">
 
         <div class="page-header">
-          <h1>CSV Validator 1.3.0</h1>
+          <h1>CSV Validator 1.4.0</h1>
         </div>
-      	<p class="lead"><code>CSV Validator</code> is a CSV validation and reporting tool which implements <a href="http://digital-preservation.github.io/csv-schema"><code>CSV Schema Language</code></a>. 
-		Released as Open Source under the <a href="http://www.mozilla.org/MPL/2.0/">Mozilla Public Licence version 2.0</a>.
-		This page is for CSV Validator 1.3.0 (and related minor releases), the equivalent page for the previous releases can now found at <a href="http://digital-preservation.github.io/csv-validator/csv-validator-1.0.html">csv-validator-1.0.html</a> and <a href="http://digital-preservation.github.io/csv-validator/csv-validator-1.1.html">csv-validator-1.1.html</a>.</p>
```

Comment:
It's my bad, one thing I should have mentioned based on what i did last time.
Change the current index.html to be `csv-validator-1.3.0.html` so it can stay there.

This sentence can then be updated to have a link to 1.3.0 documentation as well.

I am not sure, how many versions we should continue like this but that's what I did last time